manifest-generator/components/app-view.js

```diff
@@ -66,7 +70,6 @@ const pageInfo = {
       type: "string",
       fn: validations.validateDisplay,
     },
-    content: `<display-mode></display-mode>`,
```

Comment:
oh never mind, i just expanded the file.

Reply:
Yeah it looks like it was just a merge gone wrong. These duplicates were causing errors because they lacked the `value` attribute, otherwise I wouldn't have noticed the dupe at all